lists2safebrowsing.py

```diff
@@ -653,7 +667,7 @@ def start_versioning(config, chunknum, shavar_prod_lists_branches):
             )
             get_versioned_lists(config, chunknum, version=branch_name)
             print('\n*** Publish Versioned Lists ***')
-            publish_to_cloud(config, chunknum, check_versioning=True)
+            # publish_to_cloud(config, chunknum, check_versioning=True)
```

Comment:
Is this intentional?

Reply:
Corrected [44dba11](https://github.com/mozilla-services/shavar-list-creation/pull/176/commits/44dba11156d7ed7422827d3f7b75394f9ceaed92)